AppiumForMac/Server/Controller/AfMSessionController.m

```diff
@@ -891,6 +891,55 @@ -(PFUIElement*) windowForHandle:(NSString*)windowHandle
 	return [windows objectAtIndex:windowIndex];
 }
 
+-(BOOL) scrollPage:(id)element
+{
+    float increment_by = 0.25;
+    bool scroll_found = false;
+    PFUIElement* scroll;
+    float axValue = 0,adder = 0;
```

Comment:
please define variables closer to the block where they used, also each variable should be defined in its own row